### PR TITLE
[fix bug 1282842] Roll out redesigned 'forest' download page theme

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Rolls out the redesigned 'forest' download page theme as the new default.
- Moves the 'rocket' theme assets used on the home page to the `/img/home/firefox/` directory.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1282842

## Testing
- Demo server: https://www-demo1.allizom.org/en-US/firefox/new/
- Make sure both the `/new` and homepage `?v=a` pages look ok. Download page is also now being localized.

## Checklist
- [x] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
@flodolo - looks like we're going to start localizing this new download page for all locales. I'm going to put this up on a demo server shortly.

**Comment by alexgibson** [comment]
This is now up on demo1: https://www-demo1.allizom.org/firefox/new/

**Comment by alexgibson** [comment]
Ok, we have one possible string change before I can mark this as ready for l10n. Will update as soon as I have it.

File: bedrock/firefox/templates/firefox/new/horizon/base.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% add_lang_files "firefox/new" %}
+{% add_lang_files "firefox/new/horizon" %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/horizon/base.html]: @peiying2 - can you please let me know if this is an OK location for the new .lang file?

Comment (peiying2) [inline bedrock/firefox/templates/firefox/new/horizon/base.html]: @alexgibson, it seems logical.  @flodolo, any reason not to?

Comment (flodolo) [inline bedrock/firefox/templates/firefox/new/horizon/base.html]: Works for me.

**Comment by alexgibson** [comment]
OK, after several arduous days of back and forth we're managed to resolve the one line of copy.

There are now more possible page / layout change requests coming however, so for now I need to keep the blocker on this PR 😩

**Comment by alexgibson** [comment]
OK, I believe we are unblocked so I'm tentatively removing the blocked and WIP tags. This should be safe for code review.

@peiying2 - I think we are also pretty safe to consider strings final.

**Comment by alexgibson** [comment]
@peiying2 - can you please let me know if everything looks ok l10n wise?

**Comment by craigcook** [comment]
Code changes look good (it's mostly deleting stuff). r+! 😎

**Pull request merged**